Indexed_bt2/code_for_reference/create_annoted_ref.py

```diff
@@ -7,9 +7,9 @@
 from os import path as p
 
 
-def write(region, seq_chrom, annoted_ref):
+def write(seq_chrom, annoted_ref, region):
```

Comment:
bitte je ein kurzer Satz zur Dokumentation was welcher Parameter ist, z.B. annotated_ref : filepath to ...